data/attacks/stab.json

```diff
@@ -0,0 +1,174 @@
+{
+    "templates": [
+        "{target} was stabbed in the {body_parts} with a {descriptors} {weapons}!"
```

Comment:
This will show as `* gonzobot {target} was stabbed....`, it should probably be `stabs {target} in the....`

Reply:
Thanks @linuxdaemon  fixed that.